static/style.css

```
@@ -0,0 +1,7 @@
+body {
+    font-family: Arial;
```

Stylelint (17.2.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)